docs/events/rtr/com.mbta.rtr.lightrail-vehicle-information.mdx

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_label: rtr.lightrail-vehicle-updated
+---
+
+import LightRailVehicleInformation from "@site/examples/rtr-events/LightRailVehicleInformation.json";
+import EventSchemaPath from "@site/src/components/EventSchemaPath.tsx";
+import Examples from "@site/src/components/Examples.tsx";
+
+# lightrail-vehicle-updated
```

Comment:
sorry to be pedantic, perhaps `update` instead of `updated`? i.e. noun instead of past participle.

Reply:
I'm using past participle for consistency with the glides events: https://github.com/mbta/schemas/blob/main/docs/events/glides/com.mbta.ctd.glides.editors_changed.v1.mdx

I think it should either be all one tense or all the other, and it wouldn't be very convenient to change the existing events to present tense. Plus if you want to really split hairs, by the time you receive an event it has already "happened" on the producer's side so past participle could be more appropriate

Reply:
👍 gotcha! makes sense, thanks for pointing that out